src/tests/zf_apps/zfsink.c

```diff
@@ -243,14 +245,75 @@ void print_attrs(struct zf_attr* attr)
 }
 
 
-static void monitor()
+static void zf_stats_header_print(struct zf_stack* stack,
+                                  zf_stats_collection* stats_collection)
+{
+  int i;
+
+  /* only display names for the first NIC and assume others are the same */
+  if( stats_collection->num_intfs > 1 ) {
+    printf("\n#");
+  }
+  for( i = 0; i < stats_collection->layout[0]->evsl_fields_num; ++i )
+    printf("  %10s", stats_collection->layout[0]->evsl_fields[i].evsfl_name);
+}
+
+
+static void zf_stats_print(struct zf_stack* stack, uint8_t* stats_data,
+                           zf_stats_collection* stats_collection)
+{
+  int i, n, n_pad;
+  uint8_t* cur_data = stats_data;
+  const zf_stats_layout * layout;
+
+  for( n = 0; n < stats_collection->num_intfs; n++ ) {
+    if( stats_collection->num_intfs > 1 )
+      printf("\n");
```

Comment:
I think this needs to be "\n " (added space) to account for "#" in headers.  Looking at example output, values weren't aligned for multiple NIC case.